packages/react/src/components/QrCode.tsx

```diff
@@ -0,0 +1,30 @@
+import type { W3mQrCode } from '@web3modal/ui'
+import React, { useEffect } from 'react'
+
+/**
+ * Component
+ */
+export function QrCode(props: JSX.IntrinsicElements['w3m-qrcode']) {
+  const { size } = props
+
+  useEffect(() => {
+    import('@web3modal/ui')
+  })
+
+  return (
+    <div style={{ width: size, height: size }}>
```

Comment:
Does display break without the div wrapper, or could we remove it?

Reply:
It breaks since `w3m-qrcode` takes `width: 100%` and I don't want to break existing usecases for it